From 0f695e39c331ea23056ee29ac714163a0ec93c41 Mon Sep 17 00:00:00 2001
From: Taniya Das <quic_tdas@quicinc.com>
Date: Tue, 22 Oct 2024 13:36:38 +0530
Subject: [PATCH] FROMLIST: arm64: defconfig: Enable sa8775p clock controllers

Enable the SA8775P video, camera and display clock controllers to enable
the video, camera and display functionalities on Qualcomm QCS9100.

Change-Id: Iddd9265179c8192dbe5eeae6fb649f72b07d60f5
Signed-off-by: Taniya Das <quic_tdas@quicinc.com>
Link: https://lore.kernel.org/linux-arm-msm/20241022-defconfig_sa8775p_clock_controllers-v1-1-cb399b0342c8@quicinc.com/
Upstream-Status: Submitted [https://lore.kernel.org/linux-arm-msm/20241022-defconfig_sa8775p_clock_controllers-v1-1-cb399b0342c8@quicinc.com/]
---
 arch/arm64/configs/defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 93b7d01f3c78..86f28b0af2d0 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -1226,7 +1226,9 @@ CONFIG_QCM_GCC_2290=y
 CONFIG_QCM_DISPCC_2290=m
 CONFIG_QCS_GCC_404=y
 CONFIG_QCS_GCC_615=y
+CONFIG_SA_CAMCC_8775P=m
 CONFIG_QCS_GCC_8300=y
+CONFIG_SA_DISPCC_8775P=m
 CONFIG_SA_GCC_8775P=y
 CONFIG_SC_DISPCC_8280XP=m
 CONFIG_SA_GPUCC_8775P=m
@@ -1254,6 +1256,7 @@ CONFIG_SM_TCSRCC_8550=y
 CONFIG_SM_GPUCC_6115=m
 CONFIG_SM_GPUCC_8150=y
 CONFIG_SM_GPUCC_8250=y
+CONFIG_SA_VIDEOCC_8775P=m
 CONFIG_SM_VIDEOCC_8250=y
 CONFIG_QCOM_HFPLL=y
 CONFIG_CLK_GFM_LPASS_SM8250=m
-- 
2.34.1

